samples/apps/bookinfo/bookinfo.yaml

```diff
@@ -82,6 +82,34 @@ spec:
         ports:
         - containerPort: 9080
 ---
+apiVersion: extensions/v1beta1
+kind: Deployment
```

Comment:
Will e2e testing be able to load multiple services, at init time. if so then i can move it over these new services to a different file.